circle.yml

```diff
@@ -4,8 +4,7 @@ machine:
   node:
     version: v5.5.0
   environment:
-    NODE_012_VERSION: v0.12.0
-    NODE_4_VERSION: v4.2.2
+    NODE_4_VERSION: v4.5.0
```

Comment:
Yeah right now I believe it spins up 4 processes we can probably drop to 2: 4.5.0 LTS and current. It's too bad with nvm you cant do something like `nvm install latest`. I mean I guess you could do something like `nvm ls-remote | tail -1` in `setup-node`. But maybe this is out of the scope of this PR.